src/components/BaseLayerSwitcher/BaseLayerSwitcher.scss

```diff
@@ -0,0 +1,63 @@
+.rs-base-layer-switcher {
+  position: relative;
+  height: 80px;
+  width: 115px;
+  transition: 250ms width;
+  overflow: hidden;
+  display: flex;
+
+  &.rs-open {
+    width: 600px;
+  }
+
+  .rs-base-layer-switcher-button {
+    margin: 4px;
+    background-color: white;
+    border: 2px solid white;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  .rs-layer {
+    min-width: 100px;
+    background-color: rgb(187, 187, 187);
+  }
```

Comment:
This class is useless.  I still believe you should put a fixed width to the rs-base-layer-switcher-button class, otherwise if the text  is too long the div width will different between layers button.

Reply:
OK, but I need to pass in a min-width (equal to the width), otherwise the buttons will be squashed during the animation

Reply:
And also I will have go specify width (and min-width) for the close button as well

Reply:
Or I suppose I could just give that button a different class

Reply:
@oterral done, I removed the class and gave removed some more unnecessary css